README.md

```diff
@@ -1,7 +1,47 @@
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) Complete
-2. [Core Java Apps](./core_java) Complete
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Wonmin Jeong . Jarvis Consulting
```

Comment:
please run a grammar and spelling check through Grammarly if you haven't done one.